src/pages/business-page/business-page.style.js

```diff
@@ -82,6 +82,21 @@ export const useStyles = makeStyles((theme) => ({
         width: 305,
         margin: '10px 20px'
       }
+    },
+
+    '& h2': {
+      fontWeight: '400',
+      fontSize: '30px',
+      '@media (max-width: 670px)': {
+        fontSize: '21px'
+      },
+      '& span, & strong': {
```

Comment:
maybe is better to do like this: 
'& span, & strong, &h2'

Reply:
Then it's not work. I put span and strong inside h2